src/sentry/api/endpoints/organization_details.py

```diff
@@ -272,6 +370,10 @@ def save(self):
                     changed_data[key] = u"from {} to {}".format(old_val, option_inst.value)
                 option_inst.save()
 
+        trusted_realy_info = self.validated_data.get("trustedRelays")
+        if trusted_realy_info is not None:
+            self.save_trusted_relays(trusted_realy_info, changed_data, org)
```

Comment:
Please check the feature flag before saving the configuration. Otherwise, there is no more check in the codebase (and we don't want it on a per-request basis).

Reply:
feature flag is validated in `validate_trusted_relays()`... maybe I'm missing something.